elegant/gui/general_pose_annotator.py

```diff
@@ -0,0 +1,91 @@
+import freeimage
+import pathlib
+import pickle
+from PyQt5 import Qt
+import numpy
+from functools import partial
+
+from ris_widget import ris_widget
+from ris_widget import shared_resources
+
+from elegant.gui import pose_annotation
+from elegant import worm_widths
+from elegant import worm_spline
+
+from zplib.image import colorize
+
+class GeneralPoseAnnotator():
```

Comment:
if a class has no superclass you can just define it as:
```python
class Foo:
    ...
```
no need for the trailing parentheses.